### ISSUE TITLE
update cluster sorting method with better tie breakers

### ISSUE THREAD

**Issue by nathandunn** [ucscXena/XenaGoWidget#241]
under the clustering method. take a single gene TP53 for simplicity, tie break first by total hit in ONLY TP53, there will still be ties, further break by type of hits CNV vs. mutation.

**Comment by nathandunn** [comment]
Notes:
- uses clusterSort: https://github.com/ucscXena/XenaGoWidget/blob/develop/src/functions/SortFunctions.js#L71 
   - only sorts columns for given genes
   -     // sort samples first based on what gene in position 1 has the highest total
    // total is hits of +1 from all sources
    // if all are equal, then take the sum of all hit totals as a tiebreaker
    // if all are equal, then take the sum of all hit instances (all hits just equal 1) as a tiebreaker

**Comment by nathandunn** [comment]
Questions for @jingchunzhu . 
- our current method only looks at genes in the sample, so in the case you have for TP53. 

- so the next tie break CNV vs mutation (and probably type of mutation and CNV).  How would you weight these scores.  
    - CNV > mutation?
    - CNV+ > mutation3? 
    - etc? if you had to give them all numbers?  

Most of the sorting is done by this:

```
renderedData = renderedData.sort(function (a, b) {
        for (let index = 0; index < a.length; ++index) {
            if (a[index].total !== b[index].total) {
                return b[index].total - a[index].total;
            }
        }
      // tie breakers
    });
```

**Comment by nathandunn** [comment]
fixed by 6effc739b45b8256684827383ea811c7aaff8c9e
